### ISSUE TITLE
[vite-plugin-vuetify@1.0.0-alpha.17] HMR in nuxt-pages breaks auto-load of components

### ISSUE THREAD

**Issue by aleskozina** [vuetifyjs/vuetify-loader#271]
Hi,

I've made a project to reproduce the error: https://stackblitz.com/edit/nuxt-starter-ysjkep

**TLDR:**
HMR breaks the display of vuetify-components inside the pages folder.

**Longer description:**
Obviously, I don't know what exactly the error is, but after inspecting the DOM it seems that the newly loaded components after HMR do not get parsed anymore. 
This error also only appears in .vue files inside the pages folder (at least I did not find any other location but this demo project is extremely minimal)

The error disappears as soon as you manually import the used vuetify-components.

I've also made some comments inside the demo project to easily reproduce the error.

Thanks for any feedback provided!
